zerver/webhooks/hellosign/doc.md

```diff
@@ -20,7 +20,7 @@ Configuring the HelloSign integration is easy!
 
 {!congrats.md!}
 
-![](/static/images/integrations/hellosign/001.png)
+![](/static/images/integrations/Dropbox Sign/001.png)
```

Comment:
Again, we prefer to avoid spaces in filenames, just use an all-lower-case token (`dropboxsign`) here, matching the webhook path.